chia/wallet/wallet_state_manager.py

```diff
@@ -1164,14 +1198,13 @@ async def handle_clawback(
             await self.coin_store.add_coin_record(coin_record)
             # Add tx record
             created_timestamp = await self.wallet_node.get_timestamp_for_height(uint32(coin_state.created_height))
-            spend_bundle = SpendBundle([coin_spend], G2Element())
```

Comment:
I take it was unused but was it maybe supposed to be passed into the transaction record below rather than being dropped like you did with the other record above?

Reply:
The problem is this tx can be unconfirmed, and it will resend the spend bundle if it is None.  We will see double spend error in the log.